enterprise/migrations/0002_auto_20161104_0417.py

```diff
@@ -24,11 +25,26 @@ class Migration(migrations.Migration):
                 ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, verbose_name='created', editable=False)),
                 ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, verbose_name='modified', editable=False)),
                 ('logo', models.ImageField(validators=[enterprise.validators.validate_image_extension, enterprise.validators.validate_image_size], upload_to=enterprise.models.logo_path, max_length=255, blank=True, help_text='Please add only .PNG files for logo images.', null=True)),
-                ('enterprise_customer', models.OneToOneField(to='enterprise.EnterpriseCustomer')),
             ],
             options={
-                'verbose_name': 'Enterprise Customer Branding',
-                'verbose_name_plural': 'Enterprise Customer Brandings',
+                'verbose_name': 'Branding Configuration',
+                'verbose_name_plural': 'Branding Configurations',
             },
         ),
+        migrations.AddField(
+            model_name='enterprisecustomer',
```

Comment:
@asadiqbal08 That's not the right thing to do - you shouldn't merge the two migrations into one. Instead, the changes here should be added as migration 0003. Please reset this file into it's original state and create new migration.